frontend/package-lock.json

```diff
@@ -9,171 +9,171 @@
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
```

Comment:
I don't think there should be any package-lock changes since you haven't changed package.json. So maybe reset these changes/check this file out from staging?